### DISCUSSION TITLE
能否提供一个测试multipath的用例？

### DISCUSSION THREAD

**Discussion by adcen0107** [alibaba/xquic#239] [General]
能否提供一个测试multipath的用例？否则不大知道如何使用

**Comment by Kousaka-Kirino** [comment]
你好，请问现在有了吗，可否学习一下

**Reply by Yanmei-Liu** [reply]
1.参考文档编译demo client/server;
2.启动demo server：
> sudo nohup stdbuf -oL ./demo_server -p 18001 -c bbr -ld -d -M -P -s interop -f 3 &

3.用demo client测试multipath：例子中使用-V 10为draft-10版本
> ./demo_client -t 5 -A h3 -ld -L demo_clog -U "https://59.82.120.200:18001/20000000" -c p -k ckeys.log -M -i eth0 -i eth0 -V 10

其中-U参数换成你的demo_server服务器公网ip:port

**Comment by Yanmei-Liu** [comment]
1.参考文档编译demo client/server;
2.启动demo server：

sudo nohup stdbuf -oL ./demo_server -p 18001 -c bbr -ld -d -M -P -s interop -f 3 &

3.用demo client测试multipath：例子中使用-V 10为draft-10版本

./demo_client -t 5 -A h3 -ld -L demo_clog -U "https://59.82.120.200:18001/20000000" -c p -k ckeys.log -M -i eth0 -i eth0 -V 10

其中-U参数换成你的demo_server服务器公网ip:port
